src/rootcommands.h

```diff
@@ -23,7 +23,7 @@ class RootCommands {
     /** This object has no dependencies to other hlwm modules, and so the
      * 'root' pointer held by this class is just an Object-pointer.
```

Comment:
Maybe this should be updated, too

Reply:
This also makes it sound like the causalities are reversed? I'd more describe it like "This module shall have minimal dependencies, therefore the reference to root just has the Object type instead of Root."

Reply:
Both directions are 'correct': rootcommands has no dependencies, so Object suffices, and it was designed that way (on purpose).
Yes, please update it while you're at it anyway. :)

Reply:
Thanks! I updated the comment based on the suggestion by @dnnr